src/style.css

```diff
@@ -1 +1,139 @@
-/* Styles go here */
+@font-face {
+  font-family: "Roboto";
+  src: url(./fonts/Roboto-Medium.ttf);
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  justify-content: space-between;
+  align-items: center;
+  display: flex;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  /* width: 1200px;
+
+  margin: 0 auto; */
+  padding: 0 50px;
+  /* height: 60px; */
+  /* font-weight: 500px; */
+}
+
+body {
+  margin: 0;
+
+}
+
+.logo {
+  /* display: block; */
+  width: 40px;
+  height: 40px;
+  /* margin-left: 50px; */
+  /* margin-top: 10px; */
+
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+
+  margin: 0;
+  padding: 0;
+
+  /* justify-content: center; */
+  /* justify-content: center; */
+  /* margin: 0;
+  padding: 0;
+  list-style: none; */
+
+}
+
+.nav__item {
+
+  margin-right: 20px;
+
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__link {
+
+  /* white-space: nowrap; */
+
+  /* font-weight: 400px; */
+  display: flex;
+  align-items: center;
+  position: relative;
+  text-decoration: none;
+  /* font-style: normal; */
+  /* font-stretch: expanded; */
+  /* font-size: 12px;
+  line-height: 14px; */
+  text-transform: uppercase;
+  color: #000;
+  /* padding: 22.5px 0; */
+  height: 60px;
+
+}
+
+.is-active {
+  color: #00acdc;
+  /* position: relative; */
+}
+
+.nav__link::after {
+  content: string-quotes;
+  position: absolute;
+  background: #00acdc;
+  height: 4px;
+  /* width: 100%; */
+  border-radius: 8px;
+  bottom: 0;
+  left: 0;
+  right: 0;
+
+}
```

Comment:
I don't see this underline in the demo link. Also, you have failed a test about it.